dotcom-rendering/src/web/server/newsletters-page-model/defaultData.ts

```diff
@@ -0,0 +1,265 @@
+			support:
+				'https://support.theguardian.com?INTCMP=amp_header_support&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22amp_header_support%22%7D',
+			supporter:
+				'https://support.theguardian.com/subscribe?INTCMP=header_supporter_cta&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22header_supporter_cta%22%7D',
+		},
+		ampFooter: {
+			contribute:
+				'https://support.theguardian.com/contribute?INTCMP=amp_footer_support_contribute&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_FOOTER%22,%22componentId%22:%22amp_footer_support_contribute%22%7D',
+			subscribe:
+				'https://support.theguardian.com/subscribe?INTCMP=amp_footer_support_subscribe&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_FOOTER%22,%22componentId%22:%22amp_footer_support_subscribe%22%7D',
+			support:
+				'https://support.theguardian.com?INTCMP=footer_support&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_FOOTER%22,%22componentId%22:%22footer_support%22%7D',
+			supporter:
+				'https://support.theguardian.com/subscribe?INTCMP=amp_footer_supporter_cta&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_FOOTER%22,%22componentId%22:%22amp_footer_supporter_cta%22%7D',
+		},
+	},
+};
+
+export const STATIC_CONFIG: ConfigType = {
+	isPaidContent: false,
+	pageId: 'unknown-page',
+	contentType: 'static',
+	ampIframeUrl:
+		'https://assets.guim.co.uk/data/vendor/b242a49b1588bb36bdaacefe001ca77a/amp-iframe.html',
+	ajaxUrl: 'https://api.nextgen.guardianapps.co.uk',
+	shortUrlId: '/p/d8ex5',
+	switches: {},
+	keywordIds: '',
+	sharedAdTargeting: {},
+	dcrSentryDsn: 'https://1937ab71c8804b2b8438178dfdd6468f@sentry.io/1377847',
+	discussionApiUrl: 'https://discussion.theguardian.com/discussion-api',
+	sentryPublicApiKey: '344003a8d11c41d8800fbad8383fdc50',
+	commercialBundleUrl:
+		'https://assets.guim.co.uk/javascripts/bc58c17d75809551440f/graun.commercial.dcr.js',
+	discussionApiClientHeader: 'nextgen',
+	shouldHideReaderRevenue: false,
+	sentryHost: 'app.getsentry.com/35463',
+	idApiUrl: 'https://idapi.theguardian.com',
+	showRelatedContent: true,
+	adUnit: '/59666047/theguardian.com/environment/article/ng',
+	stage: 'DEV',
+	isSensitive: false,
+	revisionNumber: 'DEV',
+	section: 'environment',
+	brazeApiKey: '7f28c639-8bda-48ff-a3f6-24345abfc07c',
+	dfpAccountId: '59666047',
+	googletagUrl: '//securepubads.g.doubleclick.net/tag/js/gpt.js',
+	abTests: {},
+	edition: 'UK',
+	frontendAssetsFullURL: 'https://assets.guim.co.uk/',
+	webPublicationDate: Date.now(),
+	discussionD2Uid: 'zHoBy6HNKsk',
+	mmaUrl: 'https://manage.theguardian.com',
+};
```

Comment:
@dblatcher A lot of the data should be coming from config and not be hardcoded.

Reply:
For an actual request from frontend posted to /EmailNewsletter, all of the data comes from the request (validated as the `FENewslettersPageType` )- this hard coded version is only used at the data for the test page. 

Sorry that's not as obvious as it should be. I'll move the definition to the fixtures folder since its not for use in production.